demo/app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,7 @@
     <title>Demo</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <link href="https://unpkg.com/@primer/css-next/dist/primer.css" rel="stylesheet" />
```

Comment:
We also have this url in the docs site, right? Is there anyway we can de-dup them?

Reply:
not sure 🤔

Reply:
I think it may not be a good idea since they are "different" projects?